.github/workflows/release-please.yaml

```diff
@@ -21,38 +25,69 @@ jobs:
           command: manifest
           token: ${{secrets.GITHUB_TOKEN}}
           default-branch: main
-    outputs:
-      release_created: ${{ steps.release.outputs.release_created }}
-      release_tag_name: ${{ steps.release.outputs.tag_name }}
 
+      - name: Dump Release Please Output
+        env:
+          RELEASE_PLEASE_OUTPUT: ${{ toJson(steps.release.outputs) }}
+        run: |
+          echo "$RELEASE_PLEASE_OUTPUT"
+      - name: Determine what should be published
+        uses: actions/github-script@v6
+        id: items-to-publish
+        env:
+          CHANGED_ITEMS: '${{ steps.release.outputs.paths_released }}'
+        with:
+          script: |
```

Comment:
@beeme1mr @james-milligan might be good to have a comment summarizing what this script does.